api/s3errors/errors.go

```diff
@@ -270,7 +270,7 @@ const (
 
 	ErrPostPolicyConditionInvalidFormat
 
-	//CORS configuration errors.
+	// CORS configuration errors.
```

Comment:
Not related, btw.

Reply:
gofmt

Reply:
I know, but it's a whitespace change -> separate commit.